Sokoliuk/HW7/HW7_task_03.py

```diff
@@ -0,0 +1,14 @@
+def calculate_characters(string):
```

Comment:
good job
look at this version
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result